### DISCUSSION TITLE
why blueprint still doesn't have a grid system?

### DISCUSSION THREAD

**Discussion by kefd** [palantir/blueprint#5752] [Q&A] [answered]
#### Environment

- __Package version(s)__: 
- __Browser and OS versions__:

**Comment by wenerme** [comment]
Please do not add grid system, blueprint is about components, not styling. blueprint provide css style component, this is great, use this alot, but please let tailwind or styled to handle the infinite styling.

**Comment by adidahiya** [comment, marked as answer]
We haven't had a strong need for it, and the [Grid & dimensions docs section](https://blueprintjs.com/docs/#core/variables.grids---dimensions) still represents our current stance on this. We use flex box styling for layouts in our applications.
